powderprojection/PowderProjectionQuickView.py

```diff
@@ -0,0 +1,10 @@
+from quickview.QuickView import QuickView
+from powderprojection.PowderProjectionPresenter import PowderProjectionPresenter
+from powderprojection.MantidProjectionCalculator import MantidProjectionCalculator
+
+
+class PowderProjectionQuickView(QuickView):
+    def __init__(self,mainView,commands):
+        super(PowderProjectionQuickView,self).__init__(commands)
+        proj_calculator = MantidProjectionCalculator()
```

Comment:
`MantidProjectionCalculator` is a type which inherits from `ProjectionCalculator`. Looking through the code you never create a `mock` of this. This rings warning bells. 

The `PowderProductionPresenter` takes a `proj_calculator` as an argument. I therefore strongly suspect that you have NO tests for the `PowderProjectionPresenter`, desipite the fact that you have added it to the `QuickView`!

In future I think you should ALWAYS.

1. Write the test you need.
1. Create your presenter, calculators etc. And add the features one at a time to make the tests pass.
1. Refactor the original code to make it neater.
1. Repeat

Only when you have all your tests in place, should you consider adding corresponding `QuickViews` to support these features. DO NOT use quickviews as the basis for your testing.

Please fix this.